search-sentiments-map/public/scripts/search-topic-script.js

```diff
@@ -18,12 +18,20 @@ const MAX_SELECTED_COUNTRIES = 3;
 let countryCodeList = [];
 let countryNameList = [];
 
-/**
- * Updates the current search topic for the given country and search topic.
- */
+/** Updates the current data on the map for the user input search topic. */
 function searchTopic() {
   let topic = document.getElementById('search-topic').value;
 
+  getIsWorldLevel() ?
+      searchWorldTopic(topic) : setStateInterestsData(topic);
+}
+
+/** 
+ * Updates the current data on the map for the user-selected countries and
+ * topic at world level.
+ * @param {string} topic The topic that the user inputted.
```

Comment:
How about "entered" instead of "inputted"

Reply:
sound good